src/main/java/net/mrscauthd/beyond_earth/events/Methods.java

```diff
@@ -564,25 +545,18 @@ public static void setModelProperties(AbstractClientPlayer p_117819_, PlayerMode
             return HumanoidModel.ArmPose.EMPTY;
         } else {
             if (p_117795_.getUsedItemHand() == p_117796_ && p_117795_.getUseItemRemainingTicks() > 0) {
-                UseAnim useanim = itemstack.getUseAnimation();
-                if (useanim == UseAnim.BLOCK) {
-                    return HumanoidModel.ArmPose.BLOCK;
-                }
-
-                if (useanim == UseAnim.BOW) {
-                    return HumanoidModel.ArmPose.BOW_AND_ARROW;
-                }
-
-                if (useanim == UseAnim.SPEAR) {
-                    return HumanoidModel.ArmPose.THROW_SPEAR;
-                }
-
-                if (useanim == UseAnim.CROSSBOW && p_117796_ == p_117795_.getUsedItemHand()) {
-                    return HumanoidModel.ArmPose.CROSSBOW_CHARGE;
-                }
-
-                if (useanim == UseAnim.SPYGLASS) {
-                    return HumanoidModel.ArmPose.SPYGLASS;
+                final UseAnim useanim = itemstack.getUseAnimation();
+                switch (useanim) {
+                    case BLOCK: return HumanoidModel.ArmPose.BLOCK;
+                    case BOW: return HumanoidModel.ArmPose.BOW_AND_ARROW;
+                    case SPEAR: return HumanoidModel.ArmPose.THROW_SPEAR;
+                    case CROSSBOW: {
+                        if (p_117796_ == p_117795_.getUsedItemHand()) {
+                            return HumanoidModel.ArmPose.CROSSBOW_CHARGE;
+                        }
+                        break;
+                    }
+                    case SPYGLASS: return HumanoidModel.ArmPose.SPYGLASS;
```

Comment:
Players can't simultaneously use a spyglass and a bow, so if one condition is met, skip checking all the other conditions by using a switch.